superbench/runner/playbooks/check_env.yaml

```diff
@@ -0,0 +1,41 @@
+- name: Runtime Environment Check
```

Comment:
When Runtime Environment Check fails, does ansible run Runtime Environment Update?

Reply:
no, it will exit at fail